governance/tag-attribution.md

```diff
@@ -0,0 +1,68 @@
+# TAG Attribution Guidelines
+
+[Attribution] is the act of ascribing a work to a particular author.
+
+The TAG may be attributed with authorship of a work following a robust
+community review and approval process. Depending on the document category, a
+work will require different review processes before attribution is possible.
+
+Below is a list of works categories, and what will enable it to be attributed
+as a TAG publication.
+
+1. Publications (whitepapers, etc)
+
+- The document must follow all guidelines, protocols, and processes outlined
+    in the [publications] directory.
+
+1. Presentations (conference talks, webinars, etc)
+
+- The presentation must be made by a TAG Tech Lead or Chair
+- The presentation content must be pre-approved by a majority of TAG Leads
+- No significant concern has been left unaddressed by a TAG Tech Lead
+
+1. Blogs (On the TAG site or elsewhere)
```

Comment:
My main contention is this:

> Blogs (On the TAG site or elsewhere)
> - The Blog must be directly connected to an initiative such as a whitepaper
>   or event.
> - If the blog contains any opinions, advice, or recommendations, it follows
>   all guidelines, protocols, and processes outlined in the [publications]
>   directory.

The collective of the people here nor its leadership structure have any inherence over what anyone says anywhere else.  As Justin said on the issue:

> If it's 'by someone who happens to have some affiliation, including TAG-Security', then I don't see why the group would need to review it. This might be done as a courtesy. However, for example, NYU doesn't review everything I say or do before I do so. We're all experts in the field and should be able to comment freely as ourselves without needing consensus.

Reply:
There is a section regarding personal attribution, which covers the contention you mention. Can you highlight where it's not clear so that we can improve it?

Reply:
`or elsewhere`

Reply:
The reason this is a universal statement is because this section contains `a list of works categories, and what will enable it to be attributed as a TAG publication`. 

The following section contains guidance for how to avoid confusion when posting personally, as in Justin's example.

`There is no restriction to this, but the following examples will aid in clarity to avoid mistaken attribution.`